testbed/addons.ts

```diff
@@ -0,0 +1,19 @@
+import cdk = require('@aws-cdk/core');
+import * as eks from '@aws-cdk/aws-eks';
+import { Flux, Repository } from './addons/flux/construct';
+
+export interface AddonsProps extends cdk.StackProps {
```

Comment:
I don't think this should extend stack props.

Reply:
why so ?